examples/cfd/darcy_nested_fnos/train_nested_darcy.py

```diff
@@ -225,7 +279,7 @@ def TrainModel(cfg: DictConfig, base: SetUpInfrastructure, loaded_epoch: int) ->
                         epoch,
                     )
                     total_loss += loss * batch["darcy"].shape[0] / len(base.valid_set)
-                log.log_epoch({"Validation error": total_loss})
+                    log.log_minibatch({"Validation error": total_loss})

```

Comment:
Why log minibatch here?

Reply:
was experimenting, reverted to `log_epoch`